.circleci/config.yml

```diff
@@ -11,6 +11,23 @@ pip-run: &pip-install
     . venv/bin/activate
     pip install -q -r requirements/docs.txt
 
+image-conda-run: &image-conda-install
+  name: Install Python dependencies for figure tests
+  command: |
+    wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O $HOME/miniconda.sh
```

Comment:
why not just use the [miniconda](https://hub.docker.com/r/continuumio/miniconda3/) docker image?

Reply:
I've never used docker images, and couldn't be bothered looking up how to